client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -48,16 +49,19 @@ export class MediaManageService {
 
```

Comment:
Please unify the lists of places at the start of this class with the ones in the `mediafile.constants.ts`